subprojects/provider/src/main/kotlin/org/gradle/kotlin/dsl/provider/KotlinScriptEvaluator.kt

```diff
@@ -243,6 +251,27 @@ class StandardKotlinScriptEvaluator(
 }
 
 
+private
+fun Settings.addKotlinDevRepository() {
```

Comment:
❓ it shouldn't be necessary when not using a `-dev` or `-eap` Kotlin version.
I'm fine with handling that post-merge, a `TODO:kotlin-eap` might be nice.